.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
 install:
   - "nvm install node"
   - "nvm use node"
-  - "npm install -g eslint@^3.19.0 eslint-plugin-jquery@latest eslint-plugin-jsx-a11y@^5.0.1 eslint-plugin-import@^2.2.0 eslint-plugin-react@^7.0.1 eslint-config-airbnb"
+  - "npm install -g eslint@^3.19.0 eslint-plugin-jquery@latest eslint-plugin-jsx-a11y@^5.0.1 eslint-plugin-import@^2.2.0 eslint-plugin-react@^7.0.1"
```

Comment:
The JSX and React plugins are only installed because they are dependencies of the Airbnb config.